packages/runtime-core/src/apiCreateApp.ts

```diff
@@ -46,7 +46,7 @@ export interface App<HostElement = any> {
   mount(
     rootContainer: HostElement | string,
     isHydrate?: boolean,
-    isSVG?: boolean
+    namespace?: ElementNamespace
```

Comment:
We should add the boolean type here for public API only and normalize the namespace (convert boolean to string) and pass the normalized value for internal API.